lc-hax/Scripts/Commands/Privileged/SpawnCommand.cs

```diff
@@ -0,0 +1,67 @@
+using GameNetcodeStuff;
+using Hax;
+using System;
+using System.Collections.Generic;
+using Unity.Netcode;
+using UnityEngine;
+
+[PrivilegedCommand("/spawn")]
+public class SpawnCommand : ICommand {
+
+    private void SpawnEnemyOnPlayer(PlayerControllerB player, GameObject prefab, ulong amount = 1) {
+        if (Helper.RoundManager == null) return;
+        if (player == null) return;
+        for (ulong i = 0; i < amount; i++) {
+            GameObject enemy = UnityEngine.Object.Instantiate(prefab, player.transform.position, Quaternion.Euler(Vector3.zero));
+            if (enemy != null) {
+                if (enemy.TryGetComponent(out NetworkObject Network)) {
+                    Network.Spawn(true);
+                    if (enemy.TryGetComponent(out EnemyAI AI)) {
+                        _ = Helper.Enemies.Add(AI);
+                    }
+                }
+            }
+        }
+    }
+
+
+    public void Execute(StringArray args) {
+        if (Helper.RoundManager == null) return;
+        if (Helper.RoundManager.currentLevel == null) return;
+
+        if (args.Length < 2) {
+            Chat.Print("Usage: /spawn <player> <enemy> <amount?>");
+            return;
+        }
+
+        // Get the target player
+        PlayerControllerB? targetPlayer = Helper.GetActivePlayer(args[0]);
+        if (targetPlayer is null) {
+            Chat.Print($"Target player '{args[0]}' is not alive or found!");
+            return;
+        }
+
+        string name = args[1];
+        string EnemyName = "";
+        GameObject? prefab = null;
+        foreach (KeyValuePair<string, GameObject> enemy in Helper.SpawnableEnemies) {
+            if (enemy.Key.Contains(name, StringComparison.OrdinalIgnoreCase)) {
+                EnemyName = enemy.Key;
+                prefab = enemy.Value;
+                break;
+            }
+        }
+        if (prefab == null) {
+            return;
+        }
```

Comment:
refactor to use only Linq then you don't have to create so many separate variables.

Reply:
Both variables are needed because 
1) use the prefab
2) print the name of the enemy being spawned